crates/oro-node-semver/src/lib.rs

```diff
@@ -338,4 +377,27 @@ mod tests {
         let v = parse(ok_version);
         assert!(v.is_ok());
     }
+
+    #[derive(Serialize, Deserialize, Eq, PartialEq)]
+    struct Versioned {
+        version: Version,
+    }
+
+    #[test]
+    fn read_version_from_string() {
+        let v: Versioned = serde_json::from_str(r#"{"version":"1.2.3"}"#).unwrap();
+
+        assert_eq!(v.version, (1, 2, 3).into());
+    }
+
+    #[test]
+    fn serialize_a_version_to_string() {
+        let output = serde_json::to_string(&Versioned {
+            version: (1, 2, 3).into(),
+        })
+        .unwrap();
+        let expected: String = r#"{"version":"1.2.3"}"#.into();
+
+        assert_eq!(output, expected);
+    }
```

Comment:
could you test that build + prerelease also gets serialized/deserialized properly?